ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Tests.scala

```diff
@@ -45,6 +51,16 @@ object Tests {
     "ConfigManagementServiceIT" -> (new ConfigManagement(_)),
     "LotsOfPartiesIT" -> (new LotsOfParties(_)),
     "TransactionScaleIT" -> (new TransactionScale(_)),
+    "PerformanceEnvelope.Throughput" -> (new testtool.tests.PerformanceEnvelope.ThroughputTest(
+      logger = LoggerFactory.getLogger("PerformanceEnvelope.Throughput"),
+      envelope = Envelope.Alpha,
+      reporter = BenchmarkReporter.toFile.addReport,
+    )(_)),
+    "PerformanceEnvelope.Latency" -> (new testtool.tests.PerformanceEnvelope.LatencyTest(
+      logger = LoggerFactory.getLogger("PerformanceEnvelope.Latency"),
+      envelope = Envelope.Alpha,
+      reporter = BenchmarkReporter.toFile.addReport,
+    )(_)),
```

Comment:
It makes no sense to run these in parallel with other tests. I think we need a new category, "perf", which you specify explicitly (`--perf-tests`?). It would run the performance tests (which can be `--include`d or `--exclude`d if necessary), and force `concurrentTestRuns` to `1`.

Reply:
Yes absolutely, I'll look into that.